From: Frank Reininghaus <frank78ac@googlemail.com>
Date: Sat, 03 Oct 2015 07:26:33 +0000
Subject: Move konqpopupmenuplugin.desktop from kde-baseapps to kio
X-Git-Url: http://quickgit.kde.org/?p=kio.git&a=commitdiff&h=4b24b70c93523c5bc56c90c04a5a666331e96a1b
---
Move konqpopupmenuplugin.desktop from kde-baseapps to kio

This ensures that service menu entries are shown if the unreleased
KF5-based version of lib/konq from kde-baseapps is not installed

It would be great if the translations from lib/konq in kde-baseapps
could be merged. Thanks!

BUG: 350769
Change-Id: Id2c9a73ce95a6c8b704f3a1000c0292d1864a352
REVIEW: 125425
CCMAIL: luigi.toscano@tiscali.it
---
Slightly modified by Eric Hameleers <alien@slackware.com>
---
diff -uarN kio-5.15.0.orig/src/widgets/CMakeLists.txt kio-5.15.0/src/widgets/CMakeLists.txt
--- kio-5.15.0.orig/src/widgets/CMakeLists.txt	2015-10-05 10:11:41.000000000 +0200
+++ kio-5.15.0/src/widgets/CMakeLists.txt	2015-10-18 14:14:15.461196097 +0200
@@ -184,6 +184,7 @@
   kfileitemactionplugin.desktop
   kpropertiesdialogplugin.desktop
   kurifilterplugin.desktop
+  konqpopupmenuplugin.desktop
   kiodndpopupmenuplugin.desktop
    DESTINATION  ${KDE_INSTALL_KSERVICETYPES5DIR} )
 
diff -uarN kio-5.15.0.orig/src/widgets/konqpopupmenuplugin.desktop kio-5.15.0/src/widgets/konqpopupmenuplugin.desktop
--- kio-5.15.0.orig/src/widgets/konqpopupmenuplugin.desktop	1970-01-01 01:00:00.000000000 +0100
+++ kio-5.15.0/src/widgets/konqpopupmenuplugin.desktop	2015-10-18 14:17:09.078193339 +0200
@@ -0,0 +1,89 @@
+[Desktop Entry]
+Type=ServiceType
+X-KDE-ServiceType=KonqPopupMenu/Plugin
+Comment=Plugin for the Konqueror Popup Menu
+Comment[af]=Inplak vir die Konqueror Opspring Kieslys
+Comment[ar]=ملحق لقائمة كونكيورر المنبثقة
+Comment[as]=Konqueror প'প-আপ তালিকাৰ কাৰণে প্লাগ-ইন
+Comment[ast]=Complementu pal menú emerxente de Konqueror
+Comment[be]=Утулка для выплыўнага меню Konqueror
+Comment[be@latin]=Plugin dla vypłyŭnoha menu prahramy „Konqueror”
+Comment[bg]=Приставка за контекстното меню на браузъра
+Comment[bn]=কনকরার পপ-আপ মেনুর জন্য প্লাগ-ইন
+Comment[bn_IN]=Konqueror পপ-আপ মেনুর প্লাগ-ইন
+Comment[bs]=Priključak za K‑osvajačev iskačući meni
+Comment[ca]=Connector per al menú emergent del Konqueror
+Comment[ca@valencia]=Connector per al menú emergent del Konqueror
+Comment[cs]=Modul pro kontextovou nabídku Konqueroru
+Comment[csb]=Wtëkôcz do menu (òtmëkô knąpą mëszë) Konquerora
+Comment[cy]=Ategyn i Naidlen Konqueror
+Comment[da]=Plugin for Konquerors popop-menu
+Comment[de]=Erweiterung für das Aufklapp-Menü von Konqueror
+Comment[el]=Πρόσθετο για το αναδυόμενο μενού του Konqueror
+Comment[en_GB]=Plugin for the Konqueror Popup Menu
+Comment[eo]=Kromprogrameto por la spruĉmenuo de Konkeranto
+Comment[es]=Complemento para el menú emergente de Konqueror
+Comment[et]=Konquerori hüpikmenüü plugin
+Comment[eu]=Konquerorren laster-menuetarako plugina
+Comment[fa]=وصله برای گزینگان بالاپر Konqueror
+Comment[fi]=Konquerorin ponnahdusvalikkoliitännäinen
+Comment[fr]=Module externe pour le menu contextuel de Konqueror
+Comment[fy]=Plugin foar Konqueror's fluesmenu
+Comment[ga]=Breiseán le haghaidh Roghchláir Aníos Konqueror
+Comment[gl]=Complemento para o menú contextual de Konqueror
+Comment[gu]=કોન્કરર પોપઅપ મેનુ માટે પ્લગઇન
+Comment[he]=תוסף לתפריט המוקפץ של Konqueror
+Comment[hi]=कॉन्करर पॉपअप मेन्यू के लिए प्लगइन
+Comment[hne]=कान्करर पापअप मेन्यू बर प्लगइन
+Comment[hr]=Dodatak za Konqueror pop-up izbornik
+Comment[hsb]=zašćěpka za popup-meni w konqueroru
+Comment[hu]=Bővítőmodul a Konqueror felbukkanó menühöz
+Comment[ia]=Plugin pro le Menu de Popup de Konqueror
+Comment[id]=Plugin untuk Menu Popup Konqueror
+Comment[is]=Íforrit fyrir sprettvalmynd Konqueror Popup Menu
+Comment[it]=Estensione per il menu a comparsa di Konqueror
+Comment[ja]=Konqueror ポップアップメニューのプラグイン
+Comment[ka]=Konqueror-ის ჩამოშლადი მენიუს პლაგინი
+Comment[kk]=Konqueror баптау қалқымалы мәзір плагині
+Comment[km]=កម្មវិធី​ជំនួយ​សម្រាប់​ម៉ឺនុយ​លេច​ឡើង​របស់​ Konqueror
+Comment[kn]=ಕಾಂಕರ್ ಪುಟಿಕೆ (ಪಾಪಪ್) ಪರಿವಿಡಿಗೆ ಮಿಳಿತಾನ್ವಯ (ಪ್ಲಗಿನ್)
+Comment[ko]=Konqueror 팝업 메뉴 플러그인
+Comment[ku]=Pêveka ji bo Pêşeka Hilpekîn a Konquerorê
+Comment[lt]=Priedas pasirodančiam Konqueror meniu
+Comment[lv]=Konqueror izlecošās izvēlnes spraudnis
+Comment[mai]=कान्करर पापअप मेनू क' लेल प्लगइन
+Comment[mk]=Приклучок за контекстното мени на Konqueror
+Comment[ml]=കോണ്‍ക്വററിന്റെ പൊങ്ങിവരുന്ന മെനുവിനുള്ള സംയോജകം
+Comment[mr]=कॉन्करर पॉपअप मेन्यू करिता प्लगइन
+Comment[ms]=Plugin untuk Menu Popuo Konqueror
+Comment[nds]=Plugin för dat Konqueror-Opdukmenü
+Comment[ne]=कन्क्वेरर पपअप मेनुका लागि प्लगइन
+Comment[nl]=Plugin voor Konqueror's contextmenu
+Comment[or]=Konqueror ପପଅପ ତାଲିକା ପାଇଁ ପ୍ଲଗଇନ
+Comment[pa]=ਕੋਨਕਿਉਰੋਰ ਪਾਪਅੱਪ ਮੇਨੂ ਲਈ ਪਲੱਗਇਨ
+Comment[pl]=Wtyczka do menu (otwieranego przyciskiem myszy) Konquerora
+Comment[pt]='Plugin' para o Menu do Konqueror
+Comment[pt_BR]=Plugin do Konqueror para menus de contexto
+Comment[ro]=Extensie pentru meniul popup Konqueror
+Comment[ru]=Расширение контекстного меню Konqueror
+Comment[si]=Konqueror පොප් අප් මෙනුව සඳහා ප්ලගිනය
+Comment[sk]=Modul pre kontextové menu Konquerora
+Comment[sl]=Vstavek za Konquerorjev pojavni meni
+Comment[sr]=Прикључак за К‑освајачев искачући мени
+Comment[sr@ijekavian]=Прикључак за К‑освајачев искачући мени
+Comment[sr@ijekavianlatin]=Priključak za K‑osvajačev iskačući meni
+Comment[sr@latin]=Priključak za K‑osvajačev iskačući meni
+Comment[sv]=Insticksprogram för Konquerors popupmeny
+Comment[ta]=கான்கொரர் தோன்றும் பட்டிக்கான செருகல்கள்
+Comment[te]=Konqueror పాప్అప్ మెనూకొరకు ప్లగ్ఇన్
+Comment[tg]=Плагин барои менюи пайдошавии Konqueror
+Comment[th]=ส่วนเสริมการทำงานสำหรับเมนูผุดของคอนเควอร์เรอร์
+Comment[tr]=Konqueror Açılabilir Menüsü İçin Eklenti
+Comment[ug]=Konqueror قاڭقىش تىزىملىكىنىڭ قىستۇرمىسى
+Comment[uk]=Додаток контекстного меню Konqueror
+Comment[vi]=Trình bổ sung cho Trình đơn Bật lên của Konqueror
+Comment[wa]=Tchôke-divins po l' aspitant menu di Konqueror
+Comment[xh]=Iplagi efakiweyo ye Konqueror ye Popup Menu
+Comment[x-test]=xxPlugin for the Konqueror Popup Menuxx
+Comment[zh_CN]=Konqueror 弹出菜单插件
+Comment[zh_TW]=Konqueror 彈出式選單的外掛程式
 

